server/routes/temporary-absence-authorisations/view.njk

```diff
@@ -0,0 +1,178 @@
+              label: {
+                text: 'Latest return date'
+              },
+              value: toDate,
+              errorMessage: validationErrors | findError('toDate'),
+              classes: 'hmpps-datepicker--fixed-width govuk-body moj-search__input'
+            }) }}
+
+          </div>
+          <div class="horizontal-form">
+            {{ govukSelect({
+              label: {
+                text: "Status"
+              },
+              id: "status",
+              name: "status",
+              items: [
+                {
+                  value: '',
+                  text: "All"
+                },
+                {
+                  value: 'PENDING',
+                  text: "Pending"
+                },
+                {
+                  value: 'APPROVED',
+                  text: "Approved"
+                },
+                {
+                  value: 'DENIED',
+                  text: "Denied"
+                },
+                {
+                  value: 'WITHDRAWN',
+                  text: "Withdrawn"
+                }
+              ] | setSelectedValue(status or ''),
+              errorMessage: validationErrors | findError('status')
+            }) }}
+
+            {{ govukButton({
+              classes: "moj-search__button",
+              text: "Apply filters",
+              preventDoubleClick: true
+            }) }}
+
+            <a class="govuk-link govuk-link--no-visited-state govuk-body no-label-form-clear-link" href="?clear=true">Clear</a>
+          </div>
+        </form>
+    </div>
+
+    {% if not fromDate or not toDate %}
+      <p class="govuk-!-margin-bottom-6"><strong>Enter a date range to search and view temporary absence authorisations.</strong></p>
+    {% elseif not results.length %}
+      <p class="govuk-!-margin-bottom-6"><strong>No temporary absence authorisations match your search.</strong></p>
+    {% else %}
+      {% set rows = [] %}
+      {% for authorisation in results %}
+        {% set rows = (rows.push([
+          {
+            html: '<a class="govuk-link--no-visited-state" href="' + prisonerProfileUrl + '/prisoner/' + authorisation.person.personIdentifier + '">' + authorisation.person | firstNameSpaceLastName + "</a><br/>" + authorisation.person.personIdentifier,
+            attributes: { "data-sort-value": authorisation.person | firstNameSpaceLastName }
+          },
+          {
+            text: authorisation.fromDate | formatDate,
+            attributes: { "data-sort-value": authorisation.fromDate }
+          },
+          {
+            text: authorisation.toDate | formatDate,
+            attributes: { "data-sort-value": authorisation.toDate }
+          },
+          {
+            text: authorisation.absenceType.description,
+            attributes: { "data-sort-value": authorisation.absenceType.description }
+          },
+          {
+            text: 'Single' if authorisation.occurrenceCount === 1 else 'Repeating',
```

Comment:
This got changed to storing and returning a `repeat` property as the authorisation itself is either for single or repeat. It can still be authorised for repeat and only have one occurrence. In that instance it should still show as Repeating.

Also might be a good idea to list the occurrence count next to Repeating i.e. "Repeating {occurrenceCount} occurrences"